### PR TITLE
Added a functionality

### LINKED ISSUE #903: Add a functionality
## Expected Behavior

Add a functionality tile to improve the game.

## Possible Solution

Add a functionality and implement its behaviour.

**Implementation Plan**
 -  Create a pull request as soon as possible.
 -  Create a picture for the item you need (key/boat/...). The pictures MUST be your own work.
 - Add a tile wich fits to the item visually.
-  Add the behavior of the tile to the tiles.js file.
-  Add a tile whose behavior depends on the item.
-  Place tile and dependency into the labyrinth

### PR DESCRIPTION
**Pull request by sahilsaha7773**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-901-fossasia-labyrinth.surge.sh

Added a functionality.

- Fixes #903 

### The problem I want to solve or the facility I want to improve is/are

Add a functionality.

### Screenshots, Location of Changes

r
r2

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -404,7 +404,11 @@ const door = {
       this.ground = this.createImage("tiles/rooms/floor/Desert.svg");
     },
     visit: function() {
-         alertInfo("Let's explore the space.");
+        if (player.inventory.has('Rocket')) {
+            alertSuccess("You have a Rocket to explore space.");
```
Comment (abishekvashok) [inline js/tiles.js]: Can you properly intend these lines?

File: js/tiles.js [hunk 2]
```diff
@@ -404,7 +404,11 @@ const door = {
...
+            } else {
+             alertWarning("You can not go to the space without a rocket.");
```
Comment (abishekvashok) [inline js/tiles.js]: Here as well

File: js/tiles.js [hunk 3]
```diff
@@ -515,7 +519,8 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/blood.svg");
           },
         visit: function() {
-            alertInfo("Navy Pink");
+            player.inventory.add(['Rocket', 'Rocket.svg']);
+            alertInfo("You found a Rocket made by Sahil Saha!", "You can go to Space!!!");
```
Comment (abishekvashok) [inline js/tiles.js]: We already have an alert, so why should we add one more?

**Comment by sahilsaha7773** [comment]
The earlier alert didn't have any logic so I added a logical one

**Comment by sahilsaha7773** [comment]
I am working on the indentation

File: js/tiles.js (hunk 3 above)
Comment (sahilsaha7773) [reply js/tiles.js]: I have already removed the earlier alert

**Comment by sahilsaha7773** [comment]
Done!!

**Comment by sahilsaha7773** [comment]
@abishekvashok

File: js/tiles.js (hunk 3 above)
Comment (abishekvashok) [reply js/tiles.js]: Now, 2 alerts appear when the user visits the tile. Please see it and remove this one.

**Comment by sahilsaha7773** [comment]
are you talking about "You found a Rocket made by Sahil Saha!" and "You can go to Space!!!"? @abishekvashok

**Comment by sahilsaha7773** [comment]
Please tell

**Comment by sahilsaha7773** [comment]
Please tell where is the mistake

**Comment by YashKumarVerma** [comment]
You've mentioned him once, he'll reply when he's online. Commenting again won't help :confused:

**Comment by sahilsaha7773** [comment]
Ok sorry if I disturbed

**Comment by abishekvashok** [comment]
Sorry for the confusion, I got messed up a bit

**Comment by sahilsaha7773** [comment]
Please merge now

**Comment by abishekvashok** [comment]
Thanks

**Comment by sahilsaha7773** [comment]
Thanks for merging!!

**Pull request merged**